Formula/srcml.rb

```diff
@@ -0,0 +1,144 @@
+# Documentation: http://docs.brew.sh/Formula-Cookbook.html
+#                http://www.rubydoc.info/github/Homebrew/brew/master/Formula
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
+
+class Srcml < Formula
+  desc "srcML - an infrastructure for the exploration, analysis, and manipulation of source code"
+  homepage "http://www.srcml.org/"
+  url "http://131.123.42.38/lmcrs/beta/srcML-src.tar.gz"
+  version "0.9.5"
+  sha256 "55dd2115548e270724af4251187343656d2dfda0e7d372fee15ae27262e3fa8e"
+
+  depends_on "cmake" => :build
+  depends_on "LibArchive"
+  depends_on "libxml2"
+
+#  env do
+#    unless HOMEBREW_PREFIX.to_s == "/usr/local"
+#      ENV.append_path "HOMEBREW_LIBRARY_PATHS", "/usr/local/lib"
+#    end
+#  end
+
+  def install
+    #system "cmake", "-DINSTALL_PREFIX=/usr/local/Cellar/srcml/0.9.5", "-G", "Unix Makefiles", *std_cmake_args
+    system "cmake", *std_cmake_args
+    system "make", "install", "prefix=#{prefix}"
+  end
+
+  patch :DATA 
```

Comment:
Please submit this patch upstream and add a link to the upstream issue report in a comment.